woocommerce/woocommerce-ios
# Order detail flow: fixed UI appearance with Dark mode enabled

Fixes #1447 

I fixed all the issues on Order Detail and Order Fulfillment when dark mode is enabled (also some cell selection styles not mentioned on issue #1447). 

## TODO

- [x] Edit Order status
- [x] Processing Order: Product > Begin Fulfillment
- [x] Completed Order: Product > Details
- [x] Customer > View Billing Information

## Testing
1) Enable dark mode from device settings.
2) Navigate to an order detail.
3) Tap on every cell and check that every selection style it's ok.
4) Navigate on every sub-flow, and check that appearance of these sections it's ok. 

Update release notes:
